src/rmd_sup.erl

```diff
@@ -90,6 +90,9 @@ proxy_protobuf_config() ->
     {Ip, Port} = riak_mesos_director:proxy_protobuf_host_port(),
     [balance_protobuf, Ip, Port, 5*1000,180*1000].
 
+-spec zk_config() -> string().
```

Comment:
technically it should be `-spec zk_config() -> [[{string(), integer()}] | string()].` - I'll update.